x-pack/plugins/enterprise_search/public/applications/esre/components/esre_guide/esre_guide.tsx

```diff
@@ -11,19 +11,80 @@ import { i18n } from '@kbn/i18n';
 
 import { SetEsreChrome as SetPageChrome } from '../../../shared/kibana_chrome';
 import { EnterpriseSearchEsrePageTemplate } from '../layout/page_template';
+import {
+  EuiImage,
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiText,
+  EuiHorizontalRule,
+} from '@elastic/eui';
+import { FormattedMessage } from 'react-intl';
+
+import analyticsIllustration from '../../../../assets/images/analytics.svg';
+import scalableIllustration from '../../../../assets/images/scalable.svg';
+import simplifyIllustration from '../../../../assets/images/simplify.svg';
+import { SemanticSearchSection } from './semantic_search_section';
+import { RankAggregationSection } from './rank_aggregation_section';
+import { MeasurePerformanceSection } from './measure_performance_section';
+import { EsreDocsSection } from './esre_docs_section';
 
 export const EsreGuide: React.FC = () => {
   return (
     <EnterpriseSearchEsrePageTemplate
       restrictWidth
+      bottomBorder={false}
       pageHeader={{
         pageTitle: i18n.translate('xpack.enterpriseSearch.esre.guide.pageTitle', {
           defaultMessage: 'Enhance your search with ESRE',
         }),
       }}
     >
       <SetPageChrome />
-      <p>ESRE placeholder</p>
+      <EuiPanel color="transparent" paddingSize="none">
```

Comment:
I decided to use `EuiPanel` instead of `<EuiPageTemplate.Section>`. I had issues with padding when using the latter (for example the header is wider than the sections).